### PR TITLE
Crayon Augment

### PR DESCRIPTION
**Pull request by Ben10083**
Title

Basically took the rainbow crayon and made it a augment.

Thanks to alsoandanswer for guiding me through getting tip to change color.

### CODE REVIEW

**Comment by Ben10083** [comment]
!review

**Comment by alsoandanswer** [comment]
> If anyone who is good with sprites want to help me have it so tip changes with color selected, let me know.

Make the tip of the crayon a white overlay, re-color it appropriately, and then add the overlay to the crayon.

**Review by alsoandanswer — APPROVED** [review]

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -71,6 +71,37 @@
...
+	. = ..()
+	name = "integrated crayon"
+	update_icon()
+
+/obj/item/pen/crayon/augment/attack_self(mob/living/user as mob)
+	colour = input(user, "Please select the main colour.", "Crayon colour") as color
+	shadeColour = input(user, "Please select the shade colour.", "Crayon colour") as color
+	playsound(get_turf(src), 'sound/effects/pop.ogg', 50, 0)
+	update_icon()
+	return
```
Comment (NonQueueingMatt) [inline code/game/objects/items/crayons.dm:90]: Redundant return.

File: code/game/objects/items/crayons.dm [hunk 2]
```diff
@@ -71,6 +71,37 @@
...
+	colour = "#FFF200"
+	shadeColour = "#886422"
+	desc = "A crayon that is integrated into a user's finger. It can synthesize a multitude of colors."
+
+/obj/item/pen/crayon/augment/Initialize()
+	. = ..()
+	name = "integrated crayon"
+	update_icon()
+
+/obj/item/pen/crayon/augment/attack_self(mob/living/user as mob)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/crayons.dm:85]:
```suggestion
/obj/item/pen/crayon/augment/attack_self(mob/living/user)
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**